### ISSUE TITLE
past meetings list - SIGs are distracting 

### ISSUE THREAD

**Issue by CarlFK** [chicagopython/chipy.org#507]
https://www.chipy.org/meetings/past/

it includes SIGs, which isn't very useful.  It is accurate, and maybe someone want's to look up where a SIG was, but I think it does more harm than good.

In the context of 

> To see some examples of talks at our meetings, please visit our [YouTube](https://www.youtube.com/channel/UCT372EAC1orBOSUd2fsA8WA/videos) channel and view our [past meetings](https://www.chipy.org/meetings/past/) list.

it is concerning to see "12 Python enthusiasts attended this meeting."

**Comment by JoeJasinski** [comment]
I agree that it's confusing to see that message. Perhaps instead of removing the SIGs, we just add the title and meeting type to that list. That info is all stored in the db, so it should be able to be added. (I think there is still value in seeing a full list of all meeting)
